core/validation/initial_validation/schemas.py

```diff
@@ -105,3 +105,33 @@
         '"Future_High_Street_Fund".',
     ),
 ]
+
+PF_INITIAL_VAL_SCHEMA = [
```

Comment:
bit of a nitpick but would be good to maintain the same naming convention across these constants